src/components/Navigation.js

```diff
@@ -3,140 +3,120 @@ import styled from 'styled-components';
 
-export default function Navigation({
-  children,
-  handleResetPage,
-  handleResetPageAndShowArrow,
-  hidden,
-}) {
+export default function Navigation({ children, hidden }) {
+  const handleResetPage = useStore(state => state.handleResetPage);
+  const handleResetPageAndShowArrow = useStore(
+    state => state.handleResetPageAndShowArrow
+  );
+
   return (
     <StyledNavigation aria-label="navigation">
-      <StyledNavLinkFriends
+      <StyledNavLink
         aria-label="all non friend and friend related activities"
         to="/"
         onClick={handleResetPage}
       >
-        <img width="40" height="20" alt="friendsHomeIcon" src={friendIcon} />
-        <StyledDescription>friends</StyledDescription>
-      </StyledNavLinkFriends>
-      <StyledNavLinkGroup
+        <figure>
+          <StyledIcon
+            width="40"
+            height="20"
+            alt="friendsHomeIcon"
+            src={friendIcon}
+          />
+          <figcaption>friends</figcaption>
+        </figure>
+      </StyledNavLink>
+      <StyledNavLink
         aria-label="all group related activities"
         to="/mygroups"
         onClick={handleResetPage}
       >
-        <img width="40" height="20" alt="addAFriendIcon" src={groupIcon} />
-        <StyledDescription>groups</StyledDescription>
-      </StyledNavLinkGroup>
-      <StyledNewButton
+        <figure>
+          <StyledIcon
+            width="40"
+            height="20"
+            alt="addAFriendIcon"
+            src={groupIcon}
+          />
+          <figcaption>groups</figcaption>
+        </figure>
+      </StyledNavLink>
+      <MainNavButton
         aria-label="create a new activity"
         hidden={hidden}
         onClick={handleResetPage}
       >
         {children}
-      </StyledNewButton>
-      <StyledNavLinkActivities
+      </MainNavButton>
+      {hidden && <div />}
+      <StyledNavLink
         aria-label="all activities"
         to="/allactivities"
         onClick={handleResetPageAndShowArrow}
       >
-        <img
-          width="40"
-          height="30"
-          alt="allActivitiesIcon"
-          src={allActivitiesIcon}
-        />
-        <StyledDescription>activities</StyledDescription>
-      </StyledNavLinkActivities>
-      <StyledNavLinkInspire
+        <figure>
+          <img
+            width="40"
+            height="30"
+            alt="allActivitiesIcon"
+            src={allActivitiesIcon}
+          />
+          <figcaption>activities</figcaption>
+        </figure>
+      </StyledNavLink>
+      <StyledNavLink
         aria-label="all activities you already did"
         to="/getinspired"
         onClick={handleResetPageAndShowArrow}
       >
-        <img width="40" height="30" alt="inspireIcon" src={inspireIcon} />
-        <StyledDescription>inspiration</StyledDescription>
-      </StyledNavLinkInspire>
+        <figure>
+          <img width="40" height="30" alt="inspireIcon" src={inspireIcon} />
```

Comment:
Same alt issue here; maybe use some kind of aria attribute to make sure the user knows where the link references?

Reply:
you are totally right, this is also part of this user story to double check the accessibility.. I am actually very bad in it i think🙈